### PR TITLE
Adds a checkbox for enabling the debug script.

### PR DESCRIPTION
**Pull request by cmihail**
* By default the debug script is disabled.

Related to https://github.com/Leadfeeder/issue-tracker/issues/18381

### CODE REVIEW

**Comment by cmihail** [comment]
:white_check_mark: Review @Bajena @okulik

**Comment by cmihail** [comment]
This PR also includes the following PR which is not yet merged: https://github.com/Leadfeeder/leadfeeder-google-tag-manager-template/pull/6

To see only the new changes see https://github.com/Leadfeeder/leadfeeder-google-tag-manager-template/pull/7/commits/658d187fad5e47036218ba3a1fddbc3e91104876

File: template.tpl [hunk 1]
```diff
@@ -54,6 +54,14 @@ ___TEMPLATE_PARAMETERS___
     "defaultValue": true,
     "help": "When this checkbox is enabled the tracker will send a pageview event anytime the tag is included.\nDisabling this field might be useful e.g. when Leadfeeder tracker is used with Single Page Applications.\nYou can read more about SPA tracking \u003ca href\u003d\"https://help.leadfeeder.com/en/articles/4519005-customize-your-site-tracking-with-leadfeeder-javascript-api\" target\u003d\"_blank\"\u003eLeadfeeder Help Center\u003c/a\u003e",
     "alwaysInSummary": true
+  },
+  {
+    "type": "CHECKBOX",
+    "name": "enable_debug",
+    "checkboxText": "Enable debug script",
+    "simpleValueType": true,
+    "help": "When this checkbox is enabled the tracker adds debug logs.\nDisabling this field reduces the size of the Leadfeeder script.\n",
```
Comment (okulik) [inline template.tpl]:
```suggestion
    "help": "When this checkbox is enabled the tracker will write verbose logs to the browser's developer console.\nWe suggest turning this on only while testing your tracking script in the GTM preview mode. Disabling it reduces the size of the Leadfeeder tracking script and is a suggested option for the production websites.\n",
```

Comment (Bajena) [reply template.tpl]: Good suggestion @okulik. We could even consult the copy with Vito, but this looks good to me.

File: template.tpl [hunk 2]
```diff
@@ -54,6 +54,14 @@ ___TEMPLATE_PARAMETERS___
     "defaultValue": true,
     "help": "When this checkbox is enabled the tracker will send a pageview event anytime the tag is included.\nDisabling this field might be useful e.g. when Leadfeeder tracker is used with Single Page Applications.\nYou can read more about SPA tracking \u003ca href\u003d\"https://help.leadfeeder.com/en/articles/4519005-customize-your-site-tracking-with-leadfeeder-javascript-api\" target\u003d\"_blank\"\u003eLeadfeeder Help Center\u003c/a\u003e",
     "alwaysInSummary": true
+  },
+  {
+    "type": "CHECKBOX",
+    "name": "enable_debug",
+    "checkboxText": "Enable debug script",
```
Comment (Bajena) [inline template.tpl]: Perhaps this'd be a bit more understandable?

```suggestion
    "checkboxText": "Debug mode",
```

**Comment by cmihail** [comment]
Merged all suggestions.

**Review by okulik — APPROVED** [review]
How is this template deployed @Bajena?

**Comment by cmihail** [comment]
Update: I also added an entry in metadata.yaml as mentioned at https://developers.google.com/tag-manager/templates/gallery
I used the latest commit number as the SHA.

**Review by Bajena — APPROVED** [review]

**Comment by Bajena** [comment]
> How is this template deployed @Bajena?

This PR just needs to be merged without squashing so that the commit defined in last `metadata.yml` entry is available in `master`

**Comment by cmihail** [comment]
Could either of you merge this PR tomorrow or Friday? Unfortunately I do not have permissions for that.

**Pull request merged**